### DISCUSSION TITLE
v2.0.0 Technical Preview 7

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2190] [分享 / Share]
**功能:**

- 组件与 `v1.12.16` 同步更新.
- 新增组件 `v1 风格设置面板`.
- 在功能的管理面板中, 支持在线安装新功能.
- 搜索栏支持提供自定义选项, 目前实现的有:
  - 切换在线仓库
  - 自定义顶栏设置
- 修复 `observer.allMutations` 只处理第一个调用者.
- 修复高能进度条未固定时的样式.
- 尝试为 `bwp-video` 适配 `播放前显示封面`.
- `播放器控制栏背景色` 的不透明度选项使用 `0% ~ 100%` 做范围, 之前保存的值默认是 `0.64`, 如果你之前使用过, 更新后记得去改成 `64`.

**开发:**

- `Toast` API 支持 `Toast.mini` 并提供对应组件 `MiniToast` (基于 [tippy.js](https://atomiks.github.io/tippyjs/)), 用于在特定元素旁边弹出小提示.
- 插件允许提供描述 `PluginMetadata.description`
- 组件允许提供多语言配置和作者 `ComponentMetadata.i18n` / `ComponentMetadata.author`
- 删除所有的非必要 `eslint-disable`
- 统一内置组件的 UI 导入方式.
- Vue 内置于脚本, 不再由 `// @require` 提供.
- 组件选项支持使用滑动条提供带范围的数字选择 `ComponentOption.slider`, 例子可以参考 `registry/lib/components/video/player/control-background/index.ts`.
- 组件详情中支持扩展动作, 并内置了 `卸载` 动作.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.0-tp7'>v2.0.0 Technical Preview 7</a>.</em>

**Comment by abc2283258119** [comment]
首先，感谢开发者大佬们的努力付出~我想问一下B站的首页过滤为什么没有了呀！？

**Reply by the1812** [reply]
改名叫`简化首页`了, 和其他几个`简化xxx`的功能保持一致
原先那个`简化首页`还没做完

**Comment by sokff** [comment]
大佬 我反映一下问题，我浏览器是 百分浏览器 Chrome核心的，用v2 就没反应  ，就是打开后 左边没有小齿轮，v1正式版正常， 预览版字体乱码，但是用edge 上v2 ，又正常，看到一个问题说油猴版本低，我这油猴用测试版4.14 正式版 4.13 结果也是不出现齿轮的，以上，

**Reply by the1812** [reply]
Chrome 核心版本多少

**Reply by Geeyun-JY3** [reply]
> Chrome 核心版本多少

就是 Chromium 86.0.4240.198。这货停更半年多了。

**Comment by kethoven** [comment]
新增加的组件“视频内显示时间”的作用是什么啊？
我还以为是显示推荐视频的时长的，但是试了试，没看出什么功能啊

**Reply by the1812** [reply]
全屏时右上角显示当前系统时间

**Reply by kethoven** [reply]
> 全屏时右上角显示当前系统时间

原来如此！
谢谢！

**Reply by FoundTheWOUT** [reply]
其实一开始是想做成全屏时才显示了, 后来感觉有点麻烦, 现在做成直接显示就完事了😅

**Reply by the1812** [reply]
没做最好, 现在番剧和视频两套 CSS, 适配起来更麻烦
等我以后弄好统一的 API 再做仅全屏显示吧
